common/tests/utils.go

```diff
@@ -76,6 +76,15 @@ func CreateDsRestsMockServer(t *testing.T, testHandler restsTestHandler) (*httpt
 	return testServer, serverDetails, serviceManager
 }
 
+func CreateXscRestsMockServer(t *testing.T, testHandler restsTestHandler) (*httptest.Server, *config.ServerDetails, artifactory.ArtifactoryServicesManager) {
+	testServer := CreateRestsMockServer(testHandler)
+	serverDetails := &config.ServerDetails{Url: testServer.URL + "/", XrayUrl: testServer.URL + "/xray/"}
+
+	serviceManager, err := utils.CreateServiceManager(serverDetails, -1, 0, false)
+	assert.NoError(t, err)
+	return testServer, serverDetails, serviceManager
+}
```

Comment:
This can be created at: https://github.com/jfrog/jfrog-cli-security/pull/47
where it is used. We probably will only use mocks for Xray in our repository
added to `tests/utils/test_utils.go` / `tests/utils/test_config.go`

Reply:
all other mocks are there, so it makes sense that they will all be together in the same place.